### PR TITLE
Lint

### LINKED ISSUE #3338: `highest-rated-comment` overlap with avatar
Happens on https://github.com/ysc3839/FontMod/issues/59:
image

### PR DESCRIPTION
**Pull request by yakov116**
No description needed 🤣 

- `init: () =>` => `init()
- use pageDetect for `find-file-buffer`
- remove ajax false from `align-issue-labels`
- Fix api on gist pages

### CODE REVIEW

File: source/features/highest-rated-comment.tsx [hunk 1]
```diff
@@ -76,6 +76,8 @@ function linkBestComment(bestComment: HTMLElement): void {
 
 		// Copy avatar but link it to the comment
 		const avatar = select('.TimelineItem-avatar', bestComment)!.cloneNode(true);
+		// Remove the check icon from the preview #3338
+		select('.octicon-check.text-green', avatar)!.remove();
```
Comment (fregante) [inline source/features/highest-rated-comment.tsx:80]: This is a real bugfix, it can be its own PR

Comment (yakov116) [reply source/features/highest-rated-comment.tsx:80]: Will move later today

File: source/features/file-finder-buffer.tsx [hunk 2]
```diff
@@ -14,8 +14,8 @@ const getBufferField = onetime((): HTMLInputElement => (
 ));
 
 function pjaxStartHandler(event: CustomEvent): void {
-	const destinationURL = event.detail?.url || '';
-	if (destinationURL.split('/')[5] !== 'find') {
+	const destinationURL = new URL(event.detail?.url || location.href);
+	if (!pageDetect.isFileFinder(destinationURL)) {
```
Comment (fregante) [inline source/features/file-finder-buffer.tsx:18]: More like:
```suggestion
	const destinationURL = event.detail?.url;
	if (!destinationURL || !pageDetect.isFileFinder(new URL(destinationURL))) {
```

This skips the testing altogether when missing

File: source/features/format-conversation-titles.tsx [hunk 3]
```diff
@@ -25,5 +22,7 @@ void features.add({
 		pageDetect.isPR,
 		pageDetect.isIssue
 	],
-	init
+	init: () => {
```
Comment (fregante) [inline source/features/format-conversation-titles.tsx:25]:
```suggestion
	init() {
```

**Comment by fregante** [comment]
No need to update dependencies this often though, they're all devDependency _patches_

**Comment by yakov116** [comment]
Ok

File: source/github-helpers/api.ts [hunk 4]
```diff
@@ -55,17 +55,17 @@ export class RefinedGitHubAPIError extends Error {
 
 const settings = optionsStorage.getAll();
 
-const api3 = location.hostname === 'github.com' ?
+const api3 = ['github.com', 'gist.github.com'].includes(location.hostname) ?
 	'https://api.github.com/' :
 	`${location.origin}/api/v3/`;
-const api4 = location.hostname === 'github.com' ?
+const api4 = ['github.com', 'gist.github.com'].includes(location.hostname) ?
```
Comment (fregante) [inline source/github-helpers/api.ts:61]: We can use `isEnterprise` instead

**Comment by kidonng** [comment]
https://github.com/sindresorhus/refined-github/blob/9d2babcc1cb7f74188a0c4ee665f6229b7456dd5/source/features/linkify-user-location.tsx#L31

I think this can use `observeElement` from `helpers/simplified-element-observer`
